communication/nats_discovery/address.go

```diff
@@ -19,6 +19,13 @@ func NewAddress(server string, port int, topic string) *NatsAddress {
 	}
 }
 
+func NewBrokerAddresses(contact ContactNATSV1) *NatsAddress {
```

Comment:
We already had such factory `NewAddressForContact`, lets improve same function.
And lets improve `NewAddress(server, port, topic)` -> `NewAddress(topic, address...)`